plugins/woocommerce/client/admin/client/marketing/hooks/useIntroductionBanner.ts

```diff
@@ -30,9 +30,11 @@ export const useIntroductionBanner = (): UseIntroductionBanner => {
 			select( OPTIONS_STORE_NAME );
 
 		return {
+			// @ts-expect-error Todo: awaiting more global fix, demo: https://github.com/woocommerce/woocommerce/pull/54146
```

Comment:
I found this in the existing code base, so I just follow the same convention. See PR https://github.com/woocommerce/woocommerce/pull/54146. We can also use the PR number and link to search for more info in internal Slack and P2s.